changelog.md

```diff
@@ -1,4 +1,25 @@
 
+## 3.1.0 / 2020-09-14
+
+### New Features
+
+- A notification wall is now available in header bar. You can now seen all new activities done in Tracim.
```

Comment:
```suggestion
- A notification wall is now available in header bar. You can now see the new activities done in Tracim in real time.
```
Just a suggestion, you don't need to apply if you prefer the other way, but as we do not received the activities made by the logged user, I tried to change the "all".

Reply:
Wording change. thanks for your proposal.

Reply:
"in real time" seems good to keep, what do you think?